jupyter-lib/lib-ext/src/main/kotlin/org/jetbrains/kotlinx/jupyter/ext/graph/GraphNode.kt

```diff
@@ -0,0 +1,10 @@
+package org.jetbrains.kotlinx.jupyter.ext.graph
+
+interface GraphNode<T> {
+    val node: T
+    val label: String
+
+    val ingoingEdges: List<GraphNode<T>>
```

Comment:
Maybe incomingNodes? It will be consistent with the type

Reply:
WDYT about parentNodes/childNodes/siblingNodes? I think it's incorrect to say "incoming" about graph nodes, but I feel your concern is valid